roxie/roxiemem/roxiemem.cpp

```diff
@@ -5113,7 +5112,7 @@ class CDataBufferManager : public CInterface, implements IDataBufferManager
                 {
                     curBlock->Release();
                     curBlock = NULL;
-                    nextAddr = NULL;
+                    nextBase = NULL;
```

Comment:
I suspect it may be important NOT to clear it - in which case a comment would be good